src/components/CardHead.js

```diff
@@ -19,24 +28,74 @@ export default function CardHead({
   handleDelete,
   id,
   pathname,
+  setProfiles,
+  profiles,
 }) {
   const { on, toggle } = useToggle(false)
 
   return (
     <StyledHead onClick={toggle}>
       <StatusStyled>{status}</StatusStyled>
-      {on === false
-        ? pathname === '/profile-pool' && (
-            <DeleteSign onClick={() => handleDelete(id)}>
-              <img src={deleteIcon} alt="trashbox" />
-            </DeleteSign>
-          )
-        : ''}
-      <img src={img} alt="portrait" />
+      {on === false && pathname === '/profile-pool' && (
+        <DeleteButton onClick={() => handleDelete(id)}>
+          <Tooltip title="Delete card" placement="left">
+            <img src={deleteIcon} alt="trashbox" />
+          </Tooltip>
+        </DeleteButton>
+      )}
+      <UploadWrapper>
+        <img src={img} alt="portrait" />
+        {pathname === '/profile-pool' && (
+          <UploadButton htmlFor="imageInput">
```

Comment:
Like the DeleteButton I would create an own file where you handle the component. Then your code in this file is a bit shorter and cleaner. Maybe you can make a common component named Button and can specify it's role over props. They both have similar intern components like Tooltip. So this is a good reason to think about.

Reply:
I tried here as well to create an own component but the css of the delete function and the upload function are intertwined. I consulted Piers he said to let go of it. If you feel brave enough we can have a look together.